_posts/2024-07-12-ecrs-summary.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: single
+title:  "A Considerably Constructive Coral Conference: ECRS 2024"
+excerpt: "Presenting early-stage PhD work at Europe's largest coral conference"
+categories: conference
+toc: false
+
+---
+
+Eight trains, five countries, and a few days after leaving Cambridge I found myself in Naples, Italy, for the 2024 [European Coral Reef Symposium (ECRS)](https://ecrs2024.eu/). The European chapter of the international society, [ICRS](https://coralreefs.org/) (no prizes for guessing the acronym!), the ECRS pops up somewhere in Europe once every four years, offset by two years from ICRS. Considering timing within my PhD, location, and keynotes, it was clear that it’d be an experience to write home about.
+
+I attended ECRS to fill in the coral-loving world on my work [*predicting where coral will survive best in futures experiencing the environmental conditions caused by anthropogenic climate change*](https://orlando-code.github.io/research.html). The objective here is to guide our precious reef conservation efforts to those areas where corals are most likely to survive under climate change. This requires robust predictions of environmental suitability at spatial scales relevant to these conservation projects.
+
+While it’s quite the challenge, initial results look promising. In short, a fairly simple machine learning model is able to nicely recreate the distribution of reefs that it would expect to see, given only information about depth, and how warm, how salty, how fast-moving, how sunny (et cetera et cetera) the oceans off Australia have been over the last ~150 years. 
```

Comment:
~that it would expect to see~ -> that we see in the real world?

Reply:
It's correct as is, but clearly not clear enough: trying to communicate the model learning the relationship between environmental conditions and resulting reef.